modules/apiserversource-yaml.adoc

```diff
@@ -114,17 +122,20 @@ spec:
       containers:
         - image: quay.io/openshift-knative/knative-eventing-sources-event-display:v0.13.2
 ----
+
 +
-After you have created the `service.yaml` file, apply it by entering the following command:
+After you have created the `service.yaml` file, apply it:
 +
+
+[source,terminal]
 ----
-$ oc apply --filename service.yaml
+$ oc apply -f service.yaml
 ----
 
-. Create a trigger from the `default` broker to filter events to the service created in the previous step.
-+
-You can create the trigger by creating a file named `trigger.yaml` and copying the following sample code into it:
+. Create a trigger from the `default` broker to filter events to the service created in the previous step by creating a file named `trigger.yaml` and copying the following sample code into it:
```

Comment:
same comment about active voice ("To create a trigger...create a file named...")